### PR TITLE
Updated issue templates 

### PR DESCRIPTION
**Pull request by dilumdesilva**
I have created two issue templates, to report a bug or to request a new feature/enhancement. 

This task was listed under issue #131

### CODE REVIEW

**Review by SIMRAN1 — COMMENTED** [review]
LGTM! @jogendra please review!

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+name: Feature request
+about: Suggest an idea for Open Event ios
```
Comment (jogendra) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Change `ios` to `iOS`

File: .github/ISSUE_TEMPLATE/let-s-report-a-bug.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+<!--Provide logs for the crash here.-->
+
+**Screenshots of the issue**
+
+<!--Where-ever possible add a screenshot of the issue.-->
+
+ -   Device: [e.g. iPhone6]
+-   OS: [e.g. iOS8.1]
+
```
Comment (jogendra) [inline .github/ISSUE_TEMPLATE/let-s-report-a-bug.md:35]: Please add Xcode version too!

**Review by jogendra — CHANGES_REQUESTED** [review]
Just small changes and you are fine!

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (dilumdesilva) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Okay I will add that

File: .github/ISSUE_TEMPLATE/let-s-report-a-bug.md (hunk 2 above)
Comment (dilumdesilva) [reply .github/ISSUE_TEMPLATE/let-s-report-a-bug.md:35]: Sure

**Review by isuruAb — APPROVED** [review]

**Comment by dilumdesilva** [comment]
@jogendra updated the issue templates according to your review.

**Review by jogendra — APPROVED** [review]

**Pull request merged**